src/plugins/bar-graph/legend-area.tsx

```diff
@@ -82,9 +83,18 @@ export const LegendArea = observer(function LegendArea ({legendRef}: IProps) {
             </MenuButton>
             <Portal>
               <MenuList>
-                <MenuItem isDisabled={readOnly} onClick={() => setSecondaryAttribute(undefined)}>None</MenuItem>
```

Comment:
we have lost the None menu item

Reply:
@bgoldowsky As noted elsewhere, this is intended. "None" should no longer be an option.